admin_guide/manage_scc.adoc

```diff
@@ -0,0 +1,142 @@
+= Managing Security Context Constraints
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can manage
+link:../architecture/additional_concepts/authorization.html#security-context-constraints[security context constraints]
+in your instance via the `oc` command as a normal API objects.
+
+== Listing Security Context Constraints
+
+[options="nowrap"]
+----
+$ oc get scc
+NAME         PRIV      CAPS      HOSTDIR   SELINUX     RUNASUSER
+privileged   true      []        true      RunAsAny    RunAsAny
+restricted   false     []        false     MustRunAs   MustRunAsRange
+----
+
+== Creating New Security Context Constraints
+
+Creating a new SCC is accomplished with the `oc create` command.  First, define the SCC in a json
+or yaml file and then run `oc create` passing the file.
+
+[options="nowrap"]
+----
+{
+  "kind": "SecurityContextConstraints",
+  "apiVersion":"v1",
+
+  "metadata": {
+    "name": "scc-admin"
+  },
+  "allowPrivilegedContainer": true,
+  "runAsUser": {
+    "type": "RunAsAny"
+  },
+  "seLinuxContext": {
+    "type": "RunAsAny"
+  },
+  "groups": ["system:serviceaccounts"]
+}
+
+$ oc create -f scc_admin.json
+securitycontextconstraints/scc-admin
+$ oc get scc
+NAME         PRIV      CAPS      HOSTDIR   SELINUX     RUNASUSER
+privileged   true      []        true      RunAsAny    RunAsAny
+restricted   false     []        false     MustRunAs   MustRunAsRange
+scc-admin    true      []        false     RunAsAny    RunAsAny
+----
+
+== Deleting New Security Context Constraints
+
+[options="nowrap"]
+----
+$ oc delete scc scc-admin
+securitycontextconstraints/scc-admin
+----
```

Comment:
I tried seeing what happened when I deleted the default SCCs. It seemed that if I deleted one or the other, and restarted openshift, they did not regenerate. But if I delete both, then restarted, they both came back. Is that accurate / worth mentioning as a warning here?